src/index.ts

```diff
@@ -46,7 +46,7 @@ app.view("run-translation", async ({ ack, client, body }) => {
   });
 
   let translatedText: string | null = await deepL.translate(text, lang);
-  const pattern = /<@.+?>/g;  // /g global flag is required for replaceAll in typescript.
+  const pattern = /<@.+?>/g || /<!.+?>/g || /<#.+?>/g;  // /g global flag is required for replaceAll in typescript.
```

Comment:
You can't `||` regular `RegExp` objects in JavaScript, this needs to be done instead for it to actually work the way you expect it to work:
```suggestion
  const pattern = /(<@.+?>)|(<!.+?>)|(<#.+?>)/g; // /g global flag is required for replaceAll in typescript.
```

Reply:
Thank you so much for the suggestion @nazar-pc! I will fix it for the next deployment!